### PR TITLE
Add debug + irb gems to assist with debugging in development and tests

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
This adds the debug gem in, I'm not sure whether pry is better than debug today, but we do need something to enable interactive debugging of code in development and test.

### CODE REVIEW

**Comment by ThisIsMissEm** [comment]
@renchap @ClearlyClaire given the discussion in #27994 & discord, I've updated this PR to include the irb gem as a dependency in development and test, and referenced a specific version of debug.

**Comment by ClearlyClaire** [comment]
> I've updated this PR to include the irb gem as a dependency in development and test, and referenced a specific version of debug.

I can't see any of those changes :thinking:

**Review by ClearlyClaire — DISMISSED** [review]

File: Gemfile [hunk 1]
```diff
@@ -179,6 +179,11 @@ group :development do
 end
 
 group :development, :test do
+  # Interactive Debugging tools
+  # For why irb is in the Gemfile, see: https://ruby.social/@st0012/111444685161478182
+  gem 'debug', '~> 1.8'
+  gem 'irb', '~> 1.8'
```
Comment (renchap) [inline Gemfile:185]: I am not against having `irb` in production as well, so `rails c` benefits from its improvements

Comment (ThisIsMissEm) [reply Gemfile]: Moved!

**Review by ClearlyClaire — APPROVED** [review]

**Review by renchap — APPROVED** [review]

**Pull request merged**